builder/builder.sh

```diff
@@ -172,7 +172,7 @@ print("stage2_tag=%s" % stage2)
     local name2="${BASE_REGISTRY}/builder-base:stage2-${stage2_tag}"
 
     msg2 "Using stage-1 base ${BLU}${name1}${GRN}"
-    if ! docker run --rm --entrypoint=true "$name1"; then # skip building if the "$name1" already exists
+    if ! docker image inspect "$name1" &>/dev/null; then # skip building if the "$name1" already exists
```

Comment:
We use this pattern in builder.mk so I decided that it was sufficient. It is also faster than docker run.